resources/js/package.js

```diff
@@ -1,3 +1,16 @@
 if (import.meta.env.VITE_SENTRY_DSN !== undefined && window.config.sentry.enabled) {
-    (() => import('./sentry.js'))()
+    let deprecations = window.config.sentry.configuration.deprecations || []
+    let deprecated = false
+    deprecations.forEach((deprecation) => {
+        let test = deprecation.split('.').reduce((value, next) => value?.[next] ?? null, window)
+        if (test === null) {
+            deprecated = true
+        }
+    })
+
+    if (deprecated) {
+        console.error('This browser is not supported. Please upgrade to a newer version.')
```

Comment:
Do we want this console error?

Reply:
@indykoning? Doesn't this get reported as well?

Reply:
It wouldn't get reported as sentry will not be loaded when you get here :)